### ISSUE TITLE
Code Intel preferences not sticking - Windows

### ISSUE THREAD

**Issue by orangeaid** [Komodo/KomodoEdit#3175]
### Short Summary
Some Code Intelligence Preferences changes do not get saved

### Steps to Reproduce
Go to Edit > Preferences > Code Intelligence, and try changing the following options:
- Editing > Max files to scan without prompting
- Editing > Show calltips when typing
- Scanning > Max files to scan without prompting
Click on the "Apply" button, then on the "OK" button
Go to Edit > Preferences > Code Intelligence again

### Expected results
The changed options values/states should be visible

### Actual results
None of the changes are shown

### Platform Information

*Komodo Edit or IDE?* 
Komodo IDE
*Komodo Version?* 
11.0.0
*Operating System (and version)?* 
Windows 10 Pro 64 bit

**Comment by Naatan** [comment]
Thanks for reporting! It seems like it's only the two textfield properties in that pref screen that don't work.

**Comment by orangeaid** [comment]
Well, I cannot get the checkbox for "Show calltips when typing" to stay checked either - in my application.

**Comment by cgchoffman** [comment]
> Well, I cannot get the checkbox for "Show calltips when typing" to stay checked either - in my application.

I can't reproduce this.  Are you able to reproduce @orangeaid?

**Comment by orangeaid** [comment]
Yes - I tried it a few times before reporting, and have just re-tried it now.  If I tick the check-box for Preferences > Code Intelligence > Editing > Show calltips when typing, and save, the next time that I open the Preferences this box is not checked.

**Comment by cgchoffman** [comment]
Ok, can you restart, repro then share your logs please?

**Comment by orangeaid** [comment]
OK - here you go:
[codeintel3.log](https://github.com/Komodo/KomodoEdit/files/1341924/codeintel3.log)
[pystderr.log](https://github.com/Komodo/KomodoEdit/files/1341925/pystderr.log)

If there are any other logs, I am not sure where to find them?

**Comment by cgchoffman** [comment]
Awesome, thanks a lot @orangeaid.  Those are the logs, though the codeintel3.log wasn't' necessary, sorry I should have been more specific.  My request was ambiguous.  I should have enough info to deal with this issue.  Thanks for your help!

**Comment by cgchoffman** [comment]
Oh i was Trying `Enable Calltips` .  `Show calltips when typing` is broken.

**Comment by orangeaid** [comment]
You are very welcome.  Would it be possible for someone to let me know when this bug, and my other reported code intel bug - #3174, are fixed, and incorporated in to a fortnightly build that I can download?

**Comment by orangeaid** [comment]
Oops!  I didn't mean a forthcoming "fortnightly" build - rather a nightly build.  And, BTW, when I visit:  http://downloads.activestate.com/Komodo/nightly/komodoide/latest/  I don't see any files for v11 ?

**Comment by mitchell-as** [comment]
We haven't released a Komodo 11 nightly yet. When we do, it will be there.

**Comment by cgchoffman** [comment]
@orangeaid, the frequency of nightlies depends on what part of the development cycle we're in for which release.  As of Monday next week they will become more of an actual nightly.  Though that's not to say we will push a nightly every day.  If there were no fixes, we won't push one.  Once you install a nightly you'll be on the "nightly" update channel and able to auto-update to subsequent nightly releases.  Once 11.0.1 is released you will auto update to the "release" channel.

Long story short, look for a nightly Monday morning.  If you need it sooner I could make an effort to get it out sooner.

**Comment by cgchoffman** [comment]
https://github.com/Komodo/KomodoIDE/commit/b5d6066908885836cdfd4d033332d3df758d507f
